crates/yttrium/src/entry_point/get_sender_address.rs

```diff
@@ -60,6 +60,10 @@ where
 
     let call_builder = instance.getSenderAddress(init_code);
 
+    // Note: you may need to static call getSenderAddress() not call() as per
+    // the spec. Leaving as-is for now.
+    // let call = call_builder.call_raw().await;
```

Comment:
are you saying this is not correct and shouldn't work?

Reply:
Yeah the entry point says to do a staticcall when calling getSenderAddress. Here it's currently doing a regular call I think.

Not sure exactly the difference but it seems to work as-is for now for simple account so I'm leaving it as-is. But something to be aware of if there are issues in the future